philmccarthy/Crate
# Feature/survey genders

## Is this a fix or a feature?
- This is a feature

## What is the change/fix?
- Connects front end to back end
- Adds styleSurvey styling
- Correctly chooses gender crate based on subscription click

## Where should the reviewer start?
- StyleSurvey.js

## How should this be tested?
- Verify in postico that the back end gets updated upon completing the survey
- Verify in redux dev tools that the store gets updated upon completing the survey

## Screenshots(if appropriate)


## This PR is related to which issues:
Closes #2
